### ISSUE TITLE
Send image to PC

### ISSUE THREAD

**Issue by Dede-21** [vishnumaiea/R30X-Fingerprint-Sensor-Library#5]
Hi, thanks to write the library. I have a problem, how to send image fingerprint from arduino to PC, I hope my problem can be solve. I'm sorry my english is bad. Thank you

**Comment by UtkarshDeoli** [comment]
Yes anyone please can u help mee to (same issue)want the image in the PC
